snippets/arm-templates/deployments/with-diagnostic-setting/README.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Create or update an NGINXaaS for Azure deployment with a diagnostic setting for logging.
+languages:
+- json
+- bicep
+---
+
+# Create an NGINXaaS for Azure deployment with diagnostic setting logging
+
+### Usage
+```
+az deployment group create  --name myName  --resource-group myGroup --template-file azdeploy.json \
+    --parameters subnetName=mySubnet virtualNetworkName=myVnet publicIPName=myPublicIP capacity=50 \
+    diagnosticSettingName=myLoggingSetting \
+    diagnosticSettingStorageAccountID=/subscriptions/mysubscription/resourceGroups/myresourcegroup/providers/Microsoft.Storage/storageAccounts/mystorageaccount
+```
+
+This template provides a way to deploy a **NGINX Deployment** in a **Resource Group**. This service is still in **Public Preview**.
+
+If you're new to **NGINXaaS for Azure**, see:
+
+- [NGINXaaS for Azure Documentation](https://docs.nginx.com/nginxaas/azure/)
+- [NGINXaaS for Azure Managed Identity Documentation](https://docs.nginx.com/nginxaas/azure/management/managed-identity/) TODO: Replace all references to this broken link
```

Comment:
Does this TODO get resolved before merging?

Reply:
Yeupp...good catch

Reply:
Fixed in latest.